amxmodx/CLang.cpp

```diff
@@ -145,8 +145,8 @@ CLangMngr::CLang::CLang(const char *lang)
 {
 	m_LookUpTable.clear();
 	m_entries = 0;
-	strncpy(m_LanguageName, lang, 2);
-	m_LanguageName[2] = 0;
+	strncpy(m_LanguageName, lang, LANGNAME_LEN - 1);
```

Comment:
Looks like it'd be better to use `strcpy` instead :: https://coliru.stacked-crooked.com/a/7db05714e747cf4c